### ISSUE TITLE
No implementation found for ADO.Net object of Type Microsoft.Data.SqlClient.SqlConnectionStringBuilder

### ISSUE THREAD

**Issue by jas88** [HicServices/RDMP#877]
**Describe the bug**

Exception thrown when switching DB

ImplementationNotFoundException
No implementation found for ADO.Net object of Type Microsoft.Data.SqlClient.SqlConnectionStringBuilder

**To Reproduce**
Browsing to TEST_Catalog database on my Azure VM to fix "database needs patching" error from CI code

**Expected behavior**
Successful switch to chosen db

**Screenshots**

**RDMP Version**
v7.0.5

**Error with Stack Trace**
```
No implementation found for ADO.Net object of Type Microsoft.Data.SqlClient.SqlConnectionStringBuilder
   at FAnsi.Implementation.ImplementationManager.GetImplementation(Func`2 condition, String errorIfNotFound)
   at FAnsi.Implementation.ImplementationManager.GetImplementation(DbConnectionStringBuilder connectionStringBuilder)
   at FAnsi.Discovery.DiscoveredServer..ctor(DbConnectionStringBuilder builder)
   at Rdmp.UI.SimpleControls.ServerDatabaseTableSelector.GetDiscoveredDatabase() in D:\a\RDMP\RDMP\Rdmp.UI\SimpleControls\ServerDatabaseTableSelector.cs:line 445
   at Rdmp.UI.SimpleDialogs.ServerDatabaseTableSelectorDialog.get_SelectedDatabase() in D:\a\RDMP\RDMP\Rdmp.UI\SimpleDialogs\ServerDatabaseTableSelectorDialog.cs:line 37
   at Rdmp.UI.LocationsMenu.ChoosePlatformDatabasesUI.btnBrowseForCatalogue_Click(Object sender, EventArgs e) in D:\a\RDMP\RDMP\Rdmp.UI\LocationsMenu\ChoosePlatformDatabasesUI.cs:line 376
   at System.Windows.Forms.Control.OnClick(EventArgs e)
   at System.Windows.Forms.Button.OnClick(EventArgs e)
   at System.Windows.Forms.Button.OnMouseUp(MouseEventArgs mevent)
   at System.Windows.Forms.Control.WmMouseUp(Message& m, MouseButtons button, Int32 clicks)
   at System.Windows.Forms.Control.WndProc(Message& m)
   at System.Windows.Forms.ButtonBase.WndProc(Message& m)
   at System.Windows.Forms.Button.WndProc(Message& m)
   at System.Windows.Forms.Control.ControlNativeWindow.WndProc(Message& m)
   at System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, WM msg, IntPtr wparam, IntPtr lparam)
```

**Database Engine**
MS SQL

**Additional context**

**Comment by tznind** [comment]
Wierd, what is your current directory? are you running RDMP from outside the bin dir? Can you reproduce when running within the bin dir directly?

**Comment by tznind** [comment]
@jas88 please can you see if this PR fixes your issue (#878)
